tests/e2e/default/contacts/contact-details.wdio-spec.js

```diff
@@ -50,11 +65,16 @@ describe('Contact details page', () => {
       await updateSettings({ roles: settings.roles, permissions: settings.permissions });
     };
 
+    const DOCS_DISPLAY_LIMIT = 50;
+
     before(async () => {
       const permissions = ['can_view_contacts', 'can_view_contacts_tab', 'can_view_reports', 'can_view_tasks'];
       await updatePermissions(role, permissions);
 
-      await utils.saveDocs([parent, patient, report]);
+      await utils.saveDocs([parent, patient]);
+      await utils.saveDocs(reports);
+      await utils.saveDocs(pregnancyReport);
```

Comment:
```suggestion
      await utils.saveDocs([ ...reports, pregnancyReport ]);
```
Then, you can save all reports in 1 request.